collector_client_grpc.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
+	grpcmeta "google.golang.org/grpc/metadata"
```

Comment:
nit: You can remove the alias here and just directly access it via `metadata.NewOutgoingContext(...)`